PersonIO&Threads/src/Person.java

```diff
@@ -0,0 +1,54 @@
+public class Person {
+
+    private String name;
+    private String birthDate;
+    private PersonGender gender;
+
+    public Person(String name, String birthDate) {
+        this.name = name;
+        this.birthDate = birthDate;
+    }
+
+    @Override
+    public String toString() {
+        return "Person{" +
+                "name='" + name + '\'' +
+                ", birthDate='" + birthDate + '\'' +
+                ", gender=" + gender +
+                '}';
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getFirstName() {
+        String[] nameArr = name.split(" ");
+        return nameArr[0];
+    }
+
+    public String getBirthDate() {
+        return birthDate;
+    }
+
+    public PersonGender getGender() {
+        return gender;
+    }
+
+
+    public void setGender(String stringGender) {
+
+        if (stringGender.equalsIgnoreCase(String.valueOf(PersonGender.FEMALE))) {
```

Comment:
Logica de interpretare a gender-ului ar fi trebuit incapsulata in enum-ul `PersonGender` intro metoda scrisa de tine,